Yoast/YoastSEO.js
# 2144 set recalibration to default

## Summary

This PR can be summarized in the following changelog entry:

* The recalibrated analysis is out of its beta phase and is now the default for the SEO analysis. Thanks for testing and giving us your valuable feedback! You are awesome! 👍 

## Relevant technical choices:

* I had to adapt the full text tests to reflect the changes in the analysis. Luckily, these were only in the feedback strings and scores, and not that texts fell into different categories. E.g. where we gave back a 6 in the past, we now give back a 3 for the `subheadingKeywordAssessment`. 

## Test instructions

This PR can be tested by following these steps:

* Open a new blog post.
* Check if the analysis results reflect the changes we made in the recalibrated analysis.

Specifically, take a look at these assessments:

### Added
 * Single H1 assessment.
### Removed
 * URL stop words assessment.
 * URL length assessment.
### Changed
 * Taxonomy text length assessment 
    * Now uses the 'normal' text length assessment with different boundaries.
 * Image alt tags.
    * There is now a maximum in the number of alt tags in which your keyphrase may reside.
 * Keyword density.
    * Boundaries have changed.
    * Now has a maximum recommended value.
 * Keyphrase length assessment. 
    * Is stricter for languages with function word support (since they get stripped).
 * Outbound links assessment. 
    * Score change when no outbound links have been added: now gives a BAD (3) score instead of the previous OK (6) score.
  * Keyphrase in subheading.
    * Used to be green if there was one or more subheadings; now makes use of lower and upper boundaries.
  * Keyphrase in title
    * Function words preceding the keyphrase are filtered.

For detailed changes see: https://github.com/Yoast/YoastSEO.js/wiki/Scoring-SEO-analysis (search for "recalibrated")

Fixes #2144
